src/data/bundle.lean

```diff
@@ -32,33 +32,35 @@ def total_space := Σ x, E x
 instance [inhabited B] [inhabited (E default)] :
   inhabited (total_space E) := ⟨⟨default, default⟩⟩
 
-/-- `bundle.proj E` is the canonical projection `total_space E → B` on the base space. -/
+variables {E}
+
+/-- `bundle.proj` is the canonical projection `total_space E → B` on the base space. -/
```

Comment:
If you want to make it available to dot notation, shouldn't it be `bundle.total_space.proj`?